react/features/reactions/functions.any.js

```diff
@@ -142,3 +143,18 @@ export function getReactionsSoundsThresholds(reactions: Array<string>) {
         };
     });
 }
+
+/**
+ * Whether or not the reactions are enabled.
+ *
+ * @param {Object} state - The Redux state object.
+ * @returns {boolean}
+ */
+export function isReactionsEnabled(state: Object) {
+    if (navigator.product === 'ReactNative') {
+        return getFeatureFlag(state, REACTIONS_ENABLED, true);
```

Comment:
On mobile we should check both if the flag and config option are set. Sorry I wasn't clear on this before.